source/pi.h

```diff
@@ -16,10 +16,12 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #ifndef PI_H_
 #define PI_H_
 
+#include <numbers>
+
 
 
 // Constants to replace M_PI (which is not available on all operating systems).
```

Comment:
I think this comment can just be removed now? Or maybe an explanation of why we have PI at all